src/components/dialogs/two-windings-transformer/two-windings-transformer-creation-dialog.js

```diff
@@ -541,7 +546,7 @@ const TwoWindingsTransformerCreationDialog = ({
             setPhaseTapRows(
                 Object.values(twt.phaseTapChanger?.steps).map((step) => {
                     return {
-                        key: step.index,
+                        key: step.rho,
```

Comment:
I'm not sure rho is unique by step. It's dangerous to use it as key. Keep using the index if you have nothing else

Reply:
done